python/apps/taiga/src/taiga/stories/attachments/api.py

```diff
@@ -117,6 +120,34 @@ async def delete_story_attachment(
     await attachments_services.delete_attachment(attachment=attachment, event_on_delete=event_on_delete)
 
 
+##########################################################
+# download story attachment file
+##########################################################
+
+
+@routes.unauth_story_attachments.get(
+    "/projects/{project_id}/stories/{ref}/attachments/{attachment_id}/file/{filename}",
+    name="project.story.attachments.file",
+    summary="Download the story attachment file",
+    responses=ERROR_404 | ERROR_422,
+    response_class=StreamingResponse,
+)
+async def get_story_attachment_file(
+    project_id: B64UUID,
+    ref: int,
+    attachment_id: B64UUID,
+    filename: str,
```

Comment:
I guess this is part of the protection, as you're really accessing the storage object related to the file. Any filename will do.

I'm still missing a part here I do't fully understand.

Reply:
I dont't know if it's intended, but I also miss the Postman's endpoints related to attachments